samples/contrib/e2e-mnist/mnist-pipeline.ipynb

```diff
@@ -0,0 +1,388 @@
VdT8+xpTX0cwsbUtppWOcUSLMeQpzVtzfHwvzzjjjMpvVIxtgQUWKMaSpzGPSqURgo/f2Gjlf18i3NlSK3+fy8fWHhfTf+cW/27UtvY+z9rzWCdAgAABAgRaLyAA2XorPQkQIECAAAECBAgQINCjBVoKP+abE4LMEj4JECBAgAABAgQIEKgnUA5e5SmS6/Urb4sgUQSZyi0qnNVr9cKNOZgX/XP4p96xsS2qIT7wwANNdudKjE12/N+GCMK1tpWnnl1sscWaPay8P1eZHDhwYKUyZKODp59++mJXhBlzixBevSBe3p8/I3BVG6arnXY3943PCFuVW4Tscosqb821uL/aAGT0b693nvK83jUjCBjBpwhplZ9Bvb7lbW25hwg7Rvgph6ri3Ytp2VtqL774YktdWtwfga56rV7AtiPfw7a8f+Uqg81V0Ss/m3r3ENtOPfXUSnA41qNaX20l19pjc5itdnvtelt+T2qPbbTeFqfac0QF3FwFN4KfV155ZVHFNgKNEeTLgex4v3LAefXVVy8CoTl8F59x/8ccc0w67LDDiktEldL8GxgVImNK9O9+97tFELH82xmuEY588803i+9nTKkdocjsHcfF9z4q08b3qRx8zPcSFXojbJvbE088UVS1jPWYsrw1wdR8bFw7pomPFsfVCw7mvrWfEUyMcGMOksa1Iwyd7yW+CxEOje9suMbU8PH7Uw7dh0NLLSo75la+77wtPqM6b0wvHi0qbdZOrd2R51m+jmUCBAgQIECgbQICkG3z0psAAQIECBAgQIAAAQI9UqBR+HG3XXYu7ieHHvPN5XWVILOITwIECBAgQIAAAQIEskAO8cX64osvnjc3+xlVBmtbhIta2yJAlFsOBub12s/ytLF534ABA4pQTF7v6Odnn31WOUW9qoqVnSlVVV3M9xEBpgjptKaVg361oZ5Gx9cbUxi0to0bN67StaXnVC8o1F7veHY51FQZQGkhAmFR4S7CWuVnUOpSd7GeR7lj+R7i3nPl0egTgcvmQpn5POXKpXlbWz/b8ow68j1sy/uX34WohNnaZ9PovmtDdlGhMCo3NhdybOn9y9dqbb/cvzWfbXFq7nwRTIzpm+P+Y1rsmO769ddfLyoYlgPOUf00hx/L54uQXQQ8Y7rncjXc3Cfe0UZtrbXWKgKQce2YyrwcFI2p5qPCYwQLYxzxDp933nlFVcUItMZ09LmNHj262Bfr8T2NQHWeGj33yc83AtSxL55Jrh4bzzqPfbPNNquEPvOxOQQa6xFWjH8zosJs3Ft45fBjBDdrK5HGWA4++OB04IEHFqeLipi77LJLEQqNsUbFyqhE3FIrByDLU2nn4yKIGRUwc9t2223zYuWzM55n5WQWCBAgQIAAgVYLNP1fm60+VEcCBAgQIECAAAECBAgQ6AkCzYUfywHHHHrM95TXy33yPp8ECBAgQIAAAQIECPRegXIw5KCDDqoK+E0plfI1IwQUwZhGLcIyU7pFGCy3CLw0N54c3In+5ePy8S19lu89qqblqnItHdeR/Xka3ThHS5XTyvfXkWvGsRFUivBh+Z7L54yp1HNlxiFDhpR3NbscY4ypyBu18j3EvZcr6UUoLaYk7m6tbDQlv4f5OlHZLp5PvYBx2JSfTXNW8Rw23njjSkXDs88+u6gM2VyIr7nzdbd9eQr1euNabbXVKhUUhw0bVgQgy9VWm6uoGOHn+O2L8+cWgbxnn322qNI433zz5c1Vn+Vwb4SpywHI6Bih1qjiGn8XXHBB8Yxj+1577ZXKgdx77723si/eg5huu1GLAGL8xbVOPPHEolt5WuiLLrqo0aHF9iOOOKL4jGusuOKKqRxMjMBsvRZT3IdRBB2j4nBuK620UmEevxvNvb/ZMo6Ld7xeoPmhhx6qBDEjxFnvd78jzzOP2ScBAgQIECDQdoG+bT/EEQQIECBAgAABAgQIECDQUwRaG36MkGOuBlm+twhBxjk0AgQIECBAgAABAgQIZIFyNbynn346b27yGdXAylXqmnRow4YIt+T2zDPP5MUmnxEOipBQZ7cIx5Rbaw3imLJR+bjy+ZpbjlBPbv/5z3/yYpPPGGNLYcUmBzXYEFU2c9Atxj9x4sS6PeOaTz31VN197d0YU8w2auVpphsFvuodG2NszT1ECC8Cf+X7f+SRR+qdsrItwlkxve/UbuV3qfyO1Y6jo9/DsnNM39yotSZ4HOHHH//4x8W0wdttt11xqghWXnbZZY1O22O2xzt21FFHFVNDlyuIlm9g1KhRldUc4I3gYW7lac3ztvz55JNPFou5omKsRFDwhBNOSOeee27D97tc+TAqJTZqDz/8cLr99tuL3TF1dm3f9rzjfft2LIqQq2GWp7JurvLrmDFjivGXK75GADK3Sy65JC82+fz3v/9dqWgZU5Hna+eO48ePr7yn8Tux+eab511Vnx15nlUnskKAAAECBAi0SUAFyDZx6UyAAAECBAgQIECAAIGeI9Da8GO+o1zpMVd+zNvzet6ft/skQIAAAQIECBAgQKB3CqyyyiqVqVCvuOKKIihTWwkrpno++eSTUwSm1l133bT77rs3O31uS5IRgFx44YWL6ls333xzWmqppdLXvva1qsNiqt7TTz+9altHVsrTdkdFsXLwKO532WWXLcJ/UeksQi/f+MY3mlzuzjvvTHfddVexPfrPNttsTfq0tCGum+89rhX+yy23XJPD4llcd911aemll0577rlnu66VTxrhn3XWWSdde+21RUW1CKj98Ic/zLsrn3/5y186PXAaVehiyt9y6DUuGFPkxvS8udU+/7y93mdUf7v00kuLaXFr91955ZWVe4jpgvM0z/HeXnPNNcV1//GPf6Tvfve7tYemxx9/PP36179OUbkvgn1hP7Xa1PoexnXOP//84rbCPwKR9Z5N7tPc/X/zm98spiWOPjH1clTUi8ByTAu96qqrFtX+mju+O++LYF6e4jmqHUYFxXKL/bfddluxKQJ0uRJj
```

Comment:
This image is pretty big. Would you mind removing it from the notebook to keep the source repo smaller?

Reply:
I can do it, but the picture can show how user can get inferenceservice name well. it can help user to run this example successfully.